runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -264,10 +264,18 @@ J9::Z::CodeGenerator::initialize()
       }
 
    static bool disableIntegerToChars = (feGetEnv("TR_DisableIntegerToChars") != NULL);
-   if (cg->getSupportsVectorRegisters() && !TR::Compiler->om.canGenerateArraylets() && !TR::Compiler->om.isOffHeapAllocationEnabled() && !disableIntegerToChars && comp->target().cpu.isAtLeast(OMR_PROCESSOR_S390_Z16))
+   static bool disableIntegerStringSize = (feGetEnv("TR_DisableIntegerStringSize") != NULL);
```

Comment:
To simplify things, I think we should distinguish this two flag settings. Both have their separate evaluators. I do not see why we need them to enable / disable in same block.

Reply:
I have updated it in my latest commit.